### PR TITLE
Make dendrite run under sqlite

### PR DESCRIPTION
**Pull request by kegsay**
Should probably work... maybe.

### CODE REVIEW

**Review by anoadragon453 — APPROVED** [review]
lgtm from a functional perspective. now we just need the security review

File: dendrite/pipeline.yml
Comment (richvdh) [inline dendrite/pipeline.yml:85]: I don't know what kind of BS this is

Comment (anoadragon453) [reply dendrite/pipeline.yml:85]: https://github.com/buildkite-plugins/docker-buildkite-plugin#init-optional-boolean

> Whether or not to run an init process inside the container. This ensures that responsibilities like reaping zombie processes are performed inside the container.

Comment (richvdh) [reply dendrite/pipeline.yml:85]: Sorry, this wasn't meant to be a particular criticism of this PR - it was a throwaway comment about the state of the pipeline config in general!

But fwiw, I was referring to all three lines here. It seems like an odd way of going about things. The documentation doesn't really make it clear how these things interact with the default entrypoint. Why do we override the default entrypoint at all? Why do we disable init? And if there are good reasons for the above, why are they not written in comments?

I was going to say that we don't have any of this in the synapse pipeline config, but it turns out we do, and it's not any clearer there :/.

Anyway, it certainly doesn't need to block this PR. Just stuff that I think would be good to understand and/or clean up before we cargocult it much further.

**Pull request merged**
